src/hyp3_isce2/__main__.py

```diff
@@ -33,6 +30,9 @@ def main():
     if username and password:
         write_credentials_to_netrc_file(username, password, append=False)
```

Comment:
Ah, now that I think about it, do we want to overwrite the user's netrc file without their explicit permission? I assume most users would *probably* be supplying the same Earthdata credentials already present in their netrc file, but not sure we should make that assumption for them.

Reply:
Oh, nevermind, I didn't realize this was the hyp3 entrypoint, so it should be safe to assume that it's running in AWS?

Reply:
But if we do intend users to ever run this entrypoint locally, we should probably not automatically overwrite their netrc file.

Reply:
with `append=False`, this function will only not append to the user's netrc file, it will only create it if it doesn't exist.  See function definition [here](https://github.com/ASFHyP3/hyp3-lib/blob/b0e160d1a3c8ef42eb20a3c3cff5f6d5faedee1d/hyp3lib/fetch.py#L16).

Reply:
Ah! For some reason I thought that meant "overwrite the user's netrc file instead of appending to it".